models/general/knowledge/en/books.txt

```diff
@@ -0,0 +1,8 @@
+#Returns the name of the book and link to read more
+what is this book * |is there any book called * ? | book * | is there a book called * ?
```

Comment:
Add more possible questions to it like. 
give me a book named *
Tell me about book * 
do you know any book called * 
can you give me a book called * 
can you provide me book * 
can you provide me book called  * 
what do you know about book * 
is there any book named * 

Think of more questions also.